frontend/src/pages/SubProjects/reducer.js

```diff
@@ -137,10 +137,11 @@ export default function detailviewReducer(state = defaultState, action) {
       return state.set("isHistoryLoading", true);
     case FETCH_PROJECT_HISTORY_SUCCESS:
       return state.merge({
-        historyItems: [...state.get("historyItems"), ...fromJS(action.events)],
+        historyItems: [...fromJS(action.events), ...state.get("historyItems")],
```

Comment:
after that, `historyItems` isn't immutable, which is probably not what was intended.

```suggestion
        historyItems: fromJS(action.events).concat(state.get("historyItems")),
```